app/javascript/src/utilities/utils.js

```diff
@@ -3,7 +3,11 @@ import React from 'react'
 
 function CSRFToken ({win = window}: {win?: Window}) {
   const hasCSRFParam = win.document.head.querySelector(`meta[name='csrf-param']`)
-  const getMetaContent = meta => hasCSRFParam ? win.document.head.querySelector(`meta[name~=${meta}][content]`).content : ''
+  if (!hasCSRFParam) {
+    console.log('CSRF issue: No csrf-param present in meta tags!')
+    return null
```

Comment:
@didierofrivia what about this approach?

Reply:
It's better than empty string for sure. A few points here:

1. The constant better named it `csrfParam` without the *get* part, since it's reading a value and it's not a fn.
2. It might be ok leave it as it was, but inside a try catch block, logging the error and returning `null`.
3. It should be at least `console.warn` or maybe`error`.